packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -205,6 +215,52 @@ def generate_vip_app_permutations():
             for proxy_net in list(marathon.Network)]
 
 
+@pytest.mark.slow
+def test_ipv6(dcos_api_session):
```

Comment:
Could you add a comment here that this tests the `autoip, containerip` and *.mesos FQDN and does not test vips? Just to clarify the distinction between this and the `test_vip_ipv6` below?